packages/generator-clay-component/app/templates/_webpack.config.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const path = require('path');
+const HWP = require('html-webpack-plugin');
+
+module.exports = {
```

Comment:
hey @bryceosterhaus, as we will always repeat this for other components, maybe we can do the same thing we did in Clay v2 https://github.com/liferay/clay/tree/2.x-develop/packages/webpack-config-clay.

```javascript
const webpackCommonConfig = require('webpack-config-clay');

module.exports = Object.assign(webpackCommonConfig, {
	entry: './src/ClayBadge.js',
	output: Object.assign(webpackCommonConfig.output, {
		filename: './build/globals/clay-badge.js',
	}),
});
```
That's on Clay v2. Here it might look something like this:

```javascript
const webpackCommonConfig = require('webpack-config-clay');

module.exports = webpackCommonConfig;
```

What do you think?

Reply:
I'm not entirely sold on the idea of sharing configs through additional modules. I'm also not totally against it. I do think it might be beneficial to start out with more verbosity in the config and then if it becomes an issue later, we can abstract it.

Part of this reasoning comes from a thought of mine in that I think we to over-abstract too quickly. I personally think its easier to abstract later in the development process when you have more examples to work with.

Reply:
No problem, just a suggestion we can see how it will be in the future.